awx/main/isolated/manager.py

```diff
@@ -326,7 +326,9 @@ def health_check(self, instance_qs):
                 instance.hostname for instance in instance_qs
             ])
             self.runner_params['private_data_dir'] = private_data_dir
-            self.runner_params['forks'] = str(len(instance_qs))
```

Comment:
I don't see why we should loose the forks parameter. But perhaps it is set somewhere I do not see

Reply:
-_-